chapters/traversal.md

```diff
@@ -1,83 +1,89 @@
 # Understand Jina Recursive Document Representation
 
-In Jina, each Document is represented as a recursive representation (tree).
+This guide explains how Documents are represented within the Jina framework. It is especially relevant for understanding how segmenters, crafters and rankers work.
 
-A rooted recursive representation has a root node and every node has X children.
-In Jina, the root node is the document itself, while the *left* & *right* children are referred to as *chunks* and *matches* respectively.
+## General overview
+
+In Jina, the term Document refers to more than just a simple dataset. A Document is used for both indexing and querying. A search query is also a Document itself.
```

Comment:
remove the term `dataset`, use `more than just a simple representation for datasets` instead, since dataset is different from Jina Document in general.

Reply:
ok, thanks, I have changed that